waku/waku_relay/protocol.nim

```diff
@@ -61,7 +61,6 @@ const TopicParameters = TopicParams(
 
 # see: https://rfc.vac.dev/spec/29/#gossipsub-v10-parameters
 const GossipsubParameters = GossipSubParams.init(
-  explicit = true,
```

Comment:
What does this do?

Reply:
Good question!
This is needed to make it compile properly. Is an adaptation required due to the following commit, https://github.com/vacp2p/nim-libp2p/commit/89cad5a3ba9088cc721682469a10917903da25a0, which is out of the original scope of this PR but I needed to adapt it as well because we are bumping `nim-libp2p` with the desired commit plus that additional commit.